src/main/resources/data/exnihilosequentia/tags/items/sieves.json

```diff
@@ -0,0 +1,6 @@
+{
+  "replace": false,
+  "values": [
+    "excompressum:mana_sieve"
```

Comment:
Does mana_sieve belong here? Because a mana sieve is just a mana-powered version of the Auto Sieve.

Reply:
no, ive removed it.